hphp/test/slow/include/fast-stat-cache.php.expect

```diff
@@ -0,0 +1 @@
+OldOldNew
```

Comment:
I have add a new commit that renamed it to DelayedStatCache and add some documentation.